Dockerfile

```diff
@@ -36,7 +36,7 @@ COPY ./requirements.txt /app/requirements.txt
 
 WORKDIR /app
 
-RUN pip install maturin==0.12.3 py-sr25519-bindings==0.1.3 py-ed25519-bindings==0.1.3 py-bip39-bindings==0.1.7 wheel
+RUN pip install maturin==0.12.3 py-sr25519-bindings==0.1.4 py-ed25519-bindings==1.0.1 py-bip39-bindings==0.1.8 wheel
```

Comment:
Hmmm question here. This is dangerous. Why not make it take it from requirements.txt? I am quite sure this and requirements.tx will diverge at some point by mistake

Reply:
Didn't know it existed but maybe we can use `sys_platform == "arm64"`